easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2017.02'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
+buildopts = "all"
```

Comment:
@henkela remarks from #4718 still apply:
* use `start_dir = 'src'` rather than `prebuildopts`
* also use `buildopts = 'all'` in the `foss` easyconfig to enable building of `IMP-IO` (and update sanity check accordingly)?